scipy/optimize/optimize.py

```diff
@@ -1998,6 +1998,11 @@ def brent(func, args=(), brack=None, tol=1.48e-8, full_output=0, maxiter=500):
     Uses inverse parabolic interpolation when possible to speed up
     convergence of golden section method.
 
+    Does not ensure that the located minimum is within the initially
+    specified bracketing interval. (See 'fminbound' to ensure the same)
```

Comment:
The link to `bracket` already exists under **Parameters**. Adding it to the notes would just create a duplicate, won't it?